src/bindings/python/src/openvino/runtime/opset13/ops.py

```diff
@@ -187,3 +187,38 @@ def nms_rotated(
     }
 
     return _get_node_factory_opset13().create("NMSRotated", inputs, attributes)
+
+
+@nameable_op
+def scaled_dot_product_attention(
+    query: NodeInput,
+    key: NodeInput,
+    value: NodeInput,
+    attention_mask: NodeInput = None,
+    scale: NodeInput = None,
+    causal: bool = False,
+    name: Optional[str] = None,
+) -> Node:
+    """Return a node which implements Scaled Dot Product Attention.
+
+    Refer to https://pytorch.org/docs/stable/generated/torch.nn.functional.scaled_dot_product_attention.html for
+    detailed information.
+
+    :param query: Query tensor of shape [N, ..., L, E].
+    :param key: Key tensor of shape [N, ..., S, E]
+    :param value: Value tensor of shape [N, ..., S, Ev].
+    :param attention_mask: Optional attention mask tensor of shape [N, ..., L, S] or scalar float type zero value.
+    :param scale: Optional alternative scale, a float point type scalar
+    :param causal: If true, then autogenerates causal attention mask instead of using attention_mask input,
+                        and expects attention_mask input is not set.
+    :param name: The optional new name for output node.
```

Comment:
```suggestion
    :param query: Query tensor of shape [N, ..., L, E] and floating-point datatype.
    :param key: Key tensor of shape [N, ..., S, E] and floating-point datatype.
    :param value: Value tensor of shape [N, ..., S, Ev] and floating-point datatype.
    :param attention_mask: Optional attention mask tensor of shape [N, ..., L, S] or scalar float type zero value.
    :param scale: Optional alternative scale, a floating-point type scalar
    :param causal: If true, attention mask is autogenerated and `attention_mask` input is ignored. then autogenerates causal attention mask instead of using attention_mask input,
                        and expects attention_mask input is not set.
    :param name: The optional new name for output node.
```
Compared to specification, description for `attention_mask` is bit unclear.
Similarly for `casual`, reference specifies that input is ignored instead of expected to not be set

Reply:
Unfortunately, we cannot leave input not set in OV ops and should provide some dummy value, so it will be just ignored/not used. Please, suggest a better way if we have one. But wording here is not consistent, you are right.

>

Reply:
I think ignoring might be good enough, just mention in docs that input is ignored :)

Reply:
Already mentioned. Will just copy the description from the doc to this Python ref. Any way to share the same description between two parts? Or even three parts (including C++)?